DeeployTest/testUtils/testRunner.py

```diff
@@ -302,6 +302,8 @@ def __init__(self,
         if tiling is not argument_parser.tiling_arguments:
             raise ValueError("Specified argument parser without tile arguments for tiling test or vice versa!")
 
+        assert "CMAKE" in os.environ, "Environment variable CMAKE not set"
```

Comment:
We could also try to use `cmake` if this variable is not set.

Reply:
I'll look into how hard it is to add that change. This one was more straightforward.

Reply:
Changed here: 0d3f0128ee66fde708f4e4a1f68192550bbb2820